### PR TITLE
Active proposal styling changes and layout adjustments

### PR DESCRIPTION
**Pull request by venumasini777**
https://github.com/Plex-Engineer/canto-v3/assets/66792600/31f0efb5-4f42-4b26-9926-fa83cbec31eb

Earlier Voting options layout

After the changes:
Screenshot 2024-02-12 at 3 41 31 PM

### CODE REVIEW

File: app/governance/proposal/page.tsx
Comment (venumasini777) [inline app/governance/proposal/page.tsx:1]: Made the button disabled when the user has not selected an option and changed the text inside button

File: app/governance/components/VotingInfoBox/VotingInfoBox.tsx
Comment (venumasini777) [inline app/governance/components/VotingInfoBox/VotingInfoBox.tsx:1]: Removed radiobutton and the votes percentage info in the Voting Box as suggested by Syed

File: app/governance/components/votingChart/voteGraph.tsx [hunk 1]
```diff
@@ -434,9 +401,11 @@ export const VoteGraphBox = ({
             }}
           />
         </div>
-        <Text font="rm_mono" className={styles.tableData} size="x-sm">
-          {"ACTIVE"}
-        </Text>
+        <Container className={styles.active}>
+          <Text font="proto_mono" className={styles.tableData} size="x-sm">
+            {"ACTIVE"}
```
Comment (venumasini777) [inline app/governance/components/votingChart/voteGraph.tsx:405]: Wrapped the "Active" text inside a container to adjust the padding, margins and make it consistent with other table rows.

File: app/governance/components/votingChart/voteGraph.tsx [hunk 2]
```diff
@@ -295,106 +316,47 @@ export const VoteGraphBox = ({
...
-          }}
-        >
+        <Container height="40px" className={styles.displayVotes}>
           <Container
             className={styles.barGraph}
             direction="row"
             height="30px"
             width="44px"
           >
-            <Container
```
Comment (venumasini777) [inline app/governance/components/votingChart/voteGraph.tsx:324]: Removed redundant code and optimised the code with VoteBar component

File: app/governance/components/votingChart/voteGraph.tsx [hunk 3]
```diff
@@ -295,106 +316,47 @@ export const VoteGraphBox = ({
         return "";
     }
   };
-  const maxHeight = Math.max(
-    ...[yesHeight, noHeight, abstainHeight, vetoHeight]
```
Comment (venumasini777) [inline app/governance/components/votingChart/voteGraph.tsx:299]: Removed unnecessary Height variables used earlier. only percentages can be used

File: app/governance/components/votingChart/voteGraph.tsx [hunk 4]
```diff
@@ -263,24 +302,6 @@ export const VoteGraphBox = ({
     totalVotes > 0 ? (abstainVotes / totalVotes) * 100 : 0;
   const vetoPercentage = totalVotes > 0 ? (vetoVotes / totalVotes) * 100 : 0;
 
-  const yesHeight = (yesPercentage * size) / 150; //150 is to make the bar occupy at max 2/3rd of the total height of the container if an option get 100% votes
-  const noHeight = (noPercentage * size) / 150;
-  const abstainHeight = (abstainPercentage * size) / 150;
-  const vetoHeight = (vetoPercentage * size) / 150;
-  const getColor = (index: number): string => {
```
Comment (venumasini777) [inline app/governance/components/votingChart/voteGraph.tsx:270]: Pulled this function out of this component as it is used in VoteBar component

File: app/governance/components/votingChart/voteGraph.module.scss [hunk 5]
```diff
@@ -81,14 +81,19 @@
         }
     }
 }
-
```
Comment (venumasini777) [inline app/governance/components/votingChart/voteGraph.module.scss:84]: Made it look neat and nested

File: app/governance/components/VotingInfoBox/VotingInfoBox.module.scss [hunk 6]
```diff
@@ -35,28 +31,18 @@
...
-    height: 7px; /* Small circle size */
-    //background-color: red; /* Red circle */
+    width: 7px;
+    height: 7px; 
     border-radius: 50%; /* Make it round */
-    //margin-top: 20%;
-    //position: absolute; /* Position it inside the box */
   }
 }
-.votingInfoRow2 {
```
Comment (venumasini777) [inline app/governance/components/VotingInfoBox/VotingInfoBox.module.scss:46]: Cleaned up unnecessary CSS which was used earlier while dev

**Comment by waajito** [comment]
too many changes update your branches and then make a PR

**Pull request closed without merging**